lib/index.js

```diff
@@ -1334,10 +1349,10 @@ class Hatch {
             return this._create_hatch_archive();
         })
         .then((tarballPath) => {
-            console.log(`hatch: finished, saved to ${tarballPath}`);
+            winston.info(`hatch: finished, saved to ${tarballPath}`);
         })
         .catch((err) => {
```

Comment:
Not necessarily in scope for this pull request, but is this catch clause redundant now?

Reply:
True, we configure winston to handle exceptions. I removed it in a separate commit.